web/ui/react-app/src/Panel.tsx

```diff
@@ -157,29 +153,29 @@ class Panel extends Component<PanelProps, PanelState> {
       if (json.status !== 'success') {
         throw new Error(json.error || 'invalid response JSON');
       }
-
+      
```

Comment:
extra spaces :) Not sure wheter is IDE setting or tslint.json should be added to the project to catch such errors. @juliusv

Reply:
Yeah, I'm all for automating this stuff :) https://github.com/prometheus/prometheus/issues/6199